.github/workflows/master.yml

```diff
@@ -35,6 +35,7 @@ jobs:
     # as a cache key when caching/restoring R dependencies.
     - name: Query dependencies
       run: |
+        print(R.version)
```

Comment:
This line should additionally print out the current R version for easier debugging